report/src/problem_analysis/hci/prototyping.tex

```diff
@@ -0,0 +1,43 @@
+\section{Prototyping} %specifict prototyping
+A prototype is an instantiation of a design concept meant for describing, refining, testing, and depicting the design.\cite{BUXTON2007139_prototyping}
+
+When designing prototypes, the emphasis is not on the discovery of new knowledge and user needs, but to examine whether or not the proposed design constitutes a suitable hypothesis for the design of the product\cite{nielsen-norman-prototype-low-vs-high,BUXTON2007139_prototyping}
+That is, the design must allow for pleasurable interaction and suit user needs to a degree deemed adequate by both user and designer.
+
+In general, two categories of prototypes exist: High Fidelity (hifi) and Low Fidelity (lofi).
+Lofi prototypes are typically limited in their function and are used to emphasize concepts within the design, alternative design solutions, and model the general interaction with the artifact or system\cite{low-vs-high-fidelity-prototype}.
+These prototypes are helpful when performing ideation\cite{nielsen-norman-ideation} or when establishing user capabilities in terms of interaction with the system \cite{usefullness-of-different-prototypes,low-vs-high-fidelity-prototype}.
```

Comment:
```suggestion
These prototypes are helpful when performing ideation\cite{nielsen-norman-ideation}, or when establishing user capabilities in terms of interaction with the system \cite{usefullness-of-different-prototypes,low-vs-high-fidelity-prototype}.
```

Reply:
This does not need a comma. Only when listing 2+ things and ending with or. Same rule as with and.